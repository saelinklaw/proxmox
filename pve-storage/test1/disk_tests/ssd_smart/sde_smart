smartctl 7.1 2019-12-30 r5022 [x86_64-linux-5.4.65-1-pve] (local build)
Copyright (C) 2002-19, Bruce Allen, Christian Franke, www.smartmontools.org

=== START OF READ SMART DATA SECTION ===
SMART overall-health self-assessment test result: PASSED

SMART Attributes Data Structure revision number: 10
Vendor Specific SMART Attributes with Thresholds:
ID# ATTRIBUTE_NAME          FLAGS    VALUE WORST THRESH FAIL RAW_VALUE
  1 Raw_Read_Error_Rate     -O--CK   120   120   050    -    0/0
  5 Retired_Block_Count     PO--CK   100   100   003    -    0
  9 Power_On_Hours_and_Msec -O--CK   091   091   000    -    7963h+07m+54.620s
 12 Power_Cycle_Count       -O--CK   099   099   000    -    1153
171 Program_Fail_Count      -O-R--   100   100   000    -    0
172 Erase_Fail_Count        -O--CK   100   100   000    -    0
174 Unexpect_Power_Loss_Ct  ----CK   000   000   000    -    113
177 Wear_Range_Delta        ------   000   000   000    -    1
181 Program_Fail_Count      -O-R--   100   100   000    -    0
182 Erase_Fail_Count        -O--CK   100   100   000    -    0
187 Reported_Uncorrect      -O--C-   100   100   000    -    0
189 Airflow_Temperature_Cel ------   027   049   000    -    27 (Min/Max 2/49)
194 Temperature_Celsius     -O---K   027   049   000    -    27 (Min/Max 2/49)
195 ECC_Uncorr_Error_Count  --SRC-   120   120   000    -    0/0
196 Reallocated_Event_Count PO--CK   100   100   003    -    0
201 Unc_Soft_Read_Err_Rate  --SRC-   120   120   000    -    0/0
204 Soft_ECC_Correct_Rate   --SRC-   120   120   000    -    0/0
230 Life_Curve_Status       PO--C-   100   100   000    -    100
231 SSD_Life_Left           ------   091   091   011    -    4294967296
233 SandForce_Internal      -O--CK   000   000   000    -    6317
234 SandForce_Internal      -O--CK   000   000   000    -    4252
241 Lifetime_Writes_GiB     -O--CK   000   000   000    -    4252
242 Lifetime_Reads_GiB      -O--CK   000   000   000    -    34599
244 Unknown_Attribute       ------   099   099   010    -    4063273
                            ||||||_ K auto-keep
                            |||||__ C event count
                            ||||___ R error rate
                            |||____ S speed/performance
                            ||_____ O updated online
                            |______ P prefailure warning

