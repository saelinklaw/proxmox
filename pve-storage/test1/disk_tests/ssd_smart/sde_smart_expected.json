{
    "health": "PASSED",
    "type": "ata",
    "attributes": [
	{
	    "fail": "-",
	    "id": "  1",
	    "raw": "0/0",
	    "flags": "-O--CK",
	    "name": "Raw_Read_Error_Rate",
	    "threshold": 50,
	    "value": 120,
	    "worst": 120
	},
	{
	    "id": "  5",
	    "fail": "-",
	    "value": 100,
	    "worst": 100,
	    "threshold": 3,
	    "name": "Retired_Block_Count",
	    "flags": "PO--CK",
	    "raw": "0"
	},
	{
	    "fail": "-",
	    "id": "  9",
	    "raw": "7963h+07m+54.620s",
	    "flags": "-O--CK",
	    "worst": 91,
	    "value": 91,
	    "name": "Power_On_Hours_and_Msec",
	    "threshold": 0
	},
	{
	    "id": " 12",
	    "fail": "-",
	    "threshold": 0,
	    "name": "Power_Cycle_Count",
	    "worst": 99,
	    "value": 99,
	    "flags": "-O--CK",
	    "raw": "1153"
	},
	{
	    "flags": "-O-R--",
	    "raw": "0",
	    "worst": 100,
	    "value": 100,
	    "threshold": 0,
	    "name": "Program_Fail_Count",
	    "fail": "-",
	    "id": "171"
	},
	{
	    "fail": "-",
	    "id": "172",
	    "flags": "-O--CK",
	    "raw": "0",
	    "name": "Erase_Fail_Count",
	    "threshold": 0,
	    "worst": 100,
	    "value": 100
	},
	{
	    "fail": "-",
	    "id": "174",
	    "raw": "113",
	    "flags": "----CK",
	    "value": 0,
	    "worst": 0,
	    "threshold": 0,
	    "name": "Unexpect_Power_Loss_Ct"
	},
	{
	    "id": "177",
	    "fail": "-",
	    "value": 0,
	    "worst": 0,
	    "name": "Wear_Range_Delta",
	    "threshold": 0,
	    "flags": "------",
	    "raw": "1"
	},
	{
	    "flags": "-O-R--",
	    "raw": "0",
	    "threshold": 0,
	    "name": "Program_Fail_Count",
	    "worst": 100,
	    "value": 100,
	    "fail": "-",
	    "id": "181"
	},
	{
	    "threshold": 0,
	    "name": "Erase_Fail_Count",
	    "value": 100,
	    "worst": 100,
	    "flags": "-O--CK",
	    "raw": "0",
	    "id": "182",
	    "fail": "-"
	},
	{
	    "flags": "-O--C-",
	    "raw": "0",
	    "value": 100,
	    "worst": 100,
	    "threshold": 0,
	    "name": "Reported_Uncorrect",
	    "fail": "-",
	    "id": "187"
	},
	{
	    "value": 27,
	    "worst": 49,
	    "name": "Airflow_Temperature_Cel",
	    "threshold": 0,
	    "flags": "------",
	    "raw": "27 (Min/Max 2/49)",
	    "id": "189",
	    "fail": "-"
	},
	{
	    "threshold": 0,
	    "name": "Temperature_Celsius",
	    "worst": 49,
	    "value": 27,
	    "flags": "-O---K",
	    "raw": "27 (Min/Max 2/49)",
	    "id": "194",
	    "fail": "-"
	},
	{
	    "id": "195",
	    "fail": "-",
	    "worst": 120,
	    "value": 120,
	    "threshold": 0,
	    "name": "ECC_Uncorr_Error_Count",
	    "raw": "0/0",
	    "flags": "--SRC-"
	},
	{
	    "fail": "-",
	    "id": "196",
	    "raw": "0",
	    "flags": "PO--CK",
	    "threshold": 3,
	    "name": "Reallocated_Event_Count",
	    "value": 100,
	    "worst": 100
	},
	{
	    "value": 120,
	    "worst": 120,
	    "threshold": 0,
	    "name": "Unc_Soft_Read_Err_Rate",
	    "flags": "--SRC-",
	    "raw": "0/0",
	    "id": "201",
	    "fail": "-"
	},
	{
	    "raw": "0/0",
	    "flags": "--SRC-",
	    "value": 120,
	    "worst": 120,
	    "threshold": 0,
	    "name": "Soft_ECC_Correct_Rate",
	    "fail": "-",
	    "id": "204"
	},
	{
	    "value": 100,
	    "worst": 100,
	    "threshold": 0,
	    "name": "Life_Curve_Status",
	    "raw": "100",
	    "flags": "PO--C-",
	    "id": "230",
	    "fail": "-"
	},
	{
	    "id": "231",
	    "fail": "-",
	    "worst": 91,
	    "value": 91,
	    "name": "SSD_Life_Left",
	    "threshold": 11,
	    "flags": "------",
	    "raw": "4294967296"
	},
	{
	    "raw": "6317",
	    "flags": "-O--CK",
	    "name": "SandForce_Internal",
	    "threshold": 0,
	    "value": 0,
	    "worst": 0,
	    "fail": "-",
	    "id": "233"
	},
	{
	    "value": 0,
	    "worst": 0,
	    "name": "SandForce_Internal",
	    "threshold": 0,
	    "flags": "-O--CK",
	    "raw": "4252",
	    "id": "234",
	    "fail": "-"
	},
	{
	    "worst": 0,
	    "value": 0,
	    "name": "Lifetime_Writes_GiB",
	    "threshold": 0,
	    "flags": "-O--CK",
	    "raw": "4252",
	    "id": "241",
	    "fail": "-"
	},
	{
	    "flags": "-O--CK",
	    "raw": "34599",
	    "value": 0,
	    "worst": 0,
	    "threshold": 0,
	    "name": "Lifetime_Reads_GiB",
	    "fail": "-",
	    "id": "242"
	},
	{
	    "threshold": 10,
	    "name": "Unknown_Attribute",
	    "worst": 99,
	    "value": 99,
	    "flags": "------",
	    "raw": "4063273",
	    "id": "244",
	    "fail": "-"
	}
    ]
}
